components/climber.py

```diff
@@ -0,0 +1,31 @@
+class Climber:
+    def __init__(self) -> None:
+        self.deployed = False
+        self.retracted = False
+        self.target_speed = 0.0
+
+    def deploy(self) -> None:
+        self.target_speed = 1.0
```

Comment:
Lets turn this into a speed or voltage in a class constant so its easier to update when we get to it